Exploring_Pegasus/biclique_embedding.py

```diff
@@ -14,12 +14,15 @@
 
 import sys
 import numpy as np
-import matplotlib.pyplot as plt
 import networkx as nx
 import dwave_networkx as dnx
 from minorminer import find_embedding
 from dwave.system.samplers import DWaveSampler
 
+import matplotlib
+matplotlib.use("agg")
```

Comment:
Here's probably a better way to do this: https://github.com/dwave-examples/structural-imbalance/blob/a41490bd5e9101cf826e4b76b0eb12a46cc0fb52/structural_imbalance.py#L21 , as added by @mcfarljm

Reply:
FWIW, I used the try/except approach in the structural-imbalance code because it also has an option to do some interactive plotting that calls `plt.show()`.  In that case, `plt.show()` wouldn't work if you always set the "agg" backend.  If the code doesn't do any interactive plotting, the try/except approach might be overkill.

Reply:
That's fair. Also, once we drop python 3.5, we won't need this at all, so it probably doesn't matter.

Reply:
To keep examples future-proof, I'd suggest we do use the EAFP approach here.

Namely, after we implement a matplotlib backend for LeapIDE, we'll prefer it inside the IDE (via config file or env var, in which case hard-coding agg would override leapide).